gmplot/gmplot.py

```diff
@@ -25,7 +26,10 @@ def __init__(self, center_lat, center_lng, zoom):
         self.heatmap_points = []
         self.radpoints = []
         self.gridsetting = None
-        self.coloricon = os.path.join(os.path.dirname(__file__), 'markers/%s.png')
+        if platform.system() == 'Windows':
+            self.coloricon = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'markers\\%s.png')
```

Comment:
Is the issue the abspath, or the joiner betweeen "markers" and the marker filename? 

If it's the latter, can we can split the "markers" directory and the stub for the filename into separate arguments of os.path.join and let the platform dependence be absorbed there?

Something like
`coloricon_filename = os.path.join('markers', '%s.png')`
`coloricon = os.path.join(os.path.dirname(os.path.abspath(__file__)), coloricon_filename)`